test-requirements.txt

```diff
@@ -1,2 +1,3 @@
-molecule==2.20.1
-docker==3.7.2
\ No newline at end of file
+molecule==3.0.1
+docker==4.1.0
+ansible-lint==4.2.0
```

Comment:
you can also define here your ansible version. this way, you wont have to install separately ansible from the rest of the requirements on travis. (and its also easier to find for future changes and local tests)